apollo-router/src/plugins/headers.rs

```diff
@@ -652,7 +663,7 @@ mod test {
         let mut service = HeadersLayer::new(
             Arc::new(vec![Operation::Insert(Insert::FromBody(InsertFromBody {
                 name: "header_from_request".try_into()?,
-                path: JSONQuery::parse(".operationName")?,
+                path: JsonPathInst::from_str("$.operationName").unwrap(),
```

Comment:
Why `unwrap()` and not `?`?

Reply:
It's just a test so it's better to panic asap imo

Reply:
Ah, ok. I just spotted the change and wondered about it.